codes/classical/q-ary_digits/tensor_code.yml

```diff
@@ -0,0 +1,50 @@
+
+
+# code id, physical, logical are all lower case
+code_id: tensor_code
+physical: bits
+logical: bits
+name: 'Tensor Code'
+introduced: '\cite{doi:10.1109/ITW2.2006.323741}'
+description: | 
+  Given two linear codes \(C_1\) and \(C_2\) with parity check matrices \(H_1\) and \(H_2\) the tensor code \(C_1 \otimes C_2\) is a 
+  linear code whose parity check matrix is given by \(H_1 \otimes H_2 \) which acts on an \( n_1n_2 \)-dimensional vector space. Typically, \(C_1\) is chosen as a
+  code over \(\mathbf{F}_p\) and \(C_2\) over \(\mathbf{F}_{p^m}\) where \(p\) is a prime to make the tensor product act consistently on \(\mathbf{F}^{n_1 n_2}_{p^m}\)
```

Comment:
This is a mixup. In tensor codes both C1 and C2 are over the same alphabet, which can be any prime power. When "\(C_1\) is chosen as a  code over \(\mathbf{F}_p\) and \(C_2\) over \(\mathbf{F}_{p^m}\", then these are concatenated codes; except that tensor product does not apply in this case (let alone act consistently). Please rewrite.